Smith_Ben/Algorithms/pg50(removeblanks, acronyms).html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script type="text/javascript">
+  /*
+  $(document).ready(function(){
+    // ... your assignment here
+  });
+  */
+
+  function(remove blanks){
```

Comment:
function declaration should look like this:
`function myFunctionName(parameters){
...
}`

As you have these written they are anonymous functions and cannot be executed by calling them.  You've passed the name of your function as parameters, which will not work.